embassy-stm32/src/rcc/h.rs

```diff
@@ -194,14 +253,101 @@ pub(crate) unsafe fn init(config: Config) {
         w.set_bypass(false);
     });
 
-    #[cfg(any(pwr_h7rm0399, pwr_h7rm0455, pwr_h7rm0468))]
+    #[cfg(any(pwr_h7rm0455, pwr_h7rm0468))]
```

Comment:
SMPS is also available for those RMs, could you enable it too?

It is identical in all 3 RMs, as a general rule if a feature is identical it should be enabled in all, to avoid accumulating maintenance debt. 

If you only have rm0399 hardware it's fine, CI test has others.

Reply:
Ok. I double-checked, and all 3 RMs are identical indeed.

Currently `Direct SMPS` is hardcoded. Apparently, that's what works on Nucleos, how do you want to handle that? 
Should we make that the default on config init?

Reply:
Yes, why not. default to it on the chips that do have SMPS.